src/components/ProcessingCard/LinkButton.tsx

```diff
@@ -25,13 +27,16 @@ export default function ({
       internal={internal}
       gradientFrom="from-secondary"
       gradientTo="to-accent"
+      anchor
     >
       <Button
         small
         withArrow
         gradientFont
         type="tertiary"
-        onClick={() => onClick && onClick()}
+        onClick={() => {
+          if (cleanQuery) setLocation(location.split('#')[0])
```

Comment:
1) Clean the query params
2) Add new query param using `<a>`
3) The hook will catch the change and will scroll to the post